tests/test_convert_data_type.py

```diff
@@ -16,24 +16,25 @@
 import torch
 from parameterized import parameterized
 
+from monai.data import MetaTensor
 from monai.utils.type_conversion import convert_data_type, convert_to_dst_type
 from tests.utils import TEST_NDARRAYS
 
 TESTS: List[Tuple] = []
-for in_type in TEST_NDARRAYS + (int, float):
-    for out_type in TEST_NDARRAYS:
+for in_type in TEST_NDARRAYS + (int, float, MetaTensor):
+    for out_type in TEST_NDARRAYS + (MetaTensor,):
         TESTS.append((in_type(np.array(1.0)), out_type(np.array(1.0))))  # type: ignore
 
 TESTS_LIST: List[Tuple] = []
-for in_type in TEST_NDARRAYS + (int, float):
-    for out_type in TEST_NDARRAYS:
+for in_type in TEST_NDARRAYS + (int, float, MetaTensor):
+    for out_type in TEST_NDARRAYS + (MetaTensor,):
```

Comment:
I should have mentioned this... there are additional testing types we should include ... some renaming is needed to avoid breaking changes:
https://github.com/Project-MONAI/MONAI/blob/cb94233bc2ee924aab90ed306a8dcdad0b1e3410/tests/utils.py#L733-L736

Reply:
Sure, added in the latest commit.

Thanks.